pkg/crc/segment/segment.go

```diff
@@ -87,7 +87,7 @@ func (c *Client) Upload(ctx context.Context, action string, duration time.Durati
 		Set("duration", duration.Milliseconds()).
 		Set("tty", crcos.RunningInTerminal())
 	if err != nil {
-		properties = properties.Set("error", err.Error()).
+		properties = properties.Set("error", telemetry.SetError(err)).
```

Comment:
I think I'd remove 'err' from `Upload` arguments, and call `telemetry.SetError()` in whatever is calling `Upload`. I did not look too closely at it, so maybe it's not so great, just opening this for discussion.